pkg/repo/mysql/message.go

```diff
@@ -117,35 +118,44 @@ func (r *Repo) LoadMessages(ctx context.Context, serviceID uuid.UUID, opts commo
 
 	defer rows.Close()
 
-	messagesLookup := make(map[string][]model.Message)
+	messagesLookup := make(map[string]*model.Messages)
 
 	for rows.Next() {
 		var (
-			msg  model.Message
-			lang string
+			msg      model.Message
+			lang     string
+			original bool
 		)
 
-		if err := rows.Scan(&msg.ID, &msg.Message, &msg.Description, &msg.Fuzzy, &lang); err != nil {
+		if err := rows.Scan(&msg.ID, &msg.Message, &msg.Description, &msg.Fuzzy, &lang, &original); err != nil {
 			return nil, fmt.Errorf("repo: scan message: %w", err)
 		}
 
-		messagesLookup[lang] = append(messagesLookup[lang], msg)
+		// Lookup message by language
+		messages, ok := messagesLookup[lang]
```

Comment:
alternative: 
`		messages, found := messagesLookup[lang]
		if found {
			messages.Messages = append(messages.Messages, msg)
		} else {
			messagesLookup[lang] = &model.Messages{
				Language: language.MustParse(lang),
				Messages: []model.Message{msg},
				Original: original,
			}
		}`

Reply:
I am trying to reduce `else` statements, not big fan of them